--- origsrc/qtbase-opensource-src-5.1.1/mkspecs/features/create_cmake.prf	2014-01-01 13:22:40.961855400 -0600
+++ src/qtbase-opensource-src-5.1.1/mkspecs/features/create_cmake.prf	2014-01-02 00:12:23.430246500 -0600
@@ -105,8 +105,8 @@ mac {
     CMAKE_LIB_FILE_LOCATION_RELEASE = $${CMAKE_QT_STEM}.dll
 
     win32-g++ {
-        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}d.a
-        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.a
+        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqt5main$${QT_LIBINFIX}d.a
+        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqt5main$${QT_LIBINFIX}.a
 
         !isEmpty(CMAKE_STATIC_TYPE) {
             CMAKE_STATIC_WINDOWS_BUILD = "true"
--- origsrc/qtbase-opensource-src-5.1.1/mkspecs/features/win32/windows.prf	2014-01-01 13:22:41.183868100 -0600
+++ src/qtbase-opensource-src-5.1.1/mkspecs/features/win32/windows.prf	2014-01-02 00:13:09.920905700 -0600
@@ -4,7 +4,7 @@ contains(TEMPLATE, ".*app"){
     win32-g++:DEFINES += QT_NEEDS_QMAIN
 
     qt:for(entryLib, $$list($$unique(QMAKE_LIBS_QT_ENTRY))) {
-        isEqual(entryLib, -lqtmain): {
+        isEqual(entryLib, -lqt5main): {
             CONFIG(debug, debug|release): QMAKE_LIBS += $${entryLib}$${QT_LIBINFIX}d
             else: QMAKE_LIBS += $${entryLib}$${QT_LIBINFIX}
         } else {
--- origsrc/qtbase-opensource-src-5.1.1/src/winmain/winmain.pro	2013-08-25 13:03:31.000000000 -0500
+++ src/qtbase-opensource-src-5.1.1/src/winmain/winmain.pro	2014-01-02 00:11:25.761935400 -0600
@@ -2,7 +2,7 @@
 !win32:error("$$_FILE_ is intended only for Windows!")
 
 TEMPLATE = lib
-TARGET = qtmain
+TARGET = qt5main
 DESTDIR = $$QT.core.libs
 
 CONFIG += static
